cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -197,7 +197,8 @@ def test_create_snapshot_via_ae(request, domain, test_vm):
         execute_methods=True,
         attributes_values={"snap_name": snap_name})
 
-    wait_for(snapshot.does_snapshot_exist, timeout="2m", delay=10)
+    wait_for(lambda: snapshot.does_snapshot_exist(), timeout="2m", delay=10,
```

Comment:
excuse my missreading, i missed the addition of () - a bound method usually does fine